packages/router/src/util.ts

```diff
@@ -84,7 +84,8 @@ type SupportedRouterParamTypes = keyof typeof coreParamTypes
  *
  * route         - The route path as specified in the <Route path={...} />
  * pathname      - The pathname from the window.location.
- * allParamTypes - The object containing all param type definitions.
+ * paramTypes - The object containing all param type definitions.
+ * matchChildRoutes - Whether or not to match child routes.
```

Comment:
```suggestion
 * paramTypes       - The object containing all param type definitions.
 * matchChildRoutes - Whether or not to match child routes.
```
And also align with the params above

Reply:
Done